### ISSUE TITLE
Issues with model render

### ISSUE THREAD

**Issue by siddhartpai** [artoolkit/jsartoolkit5#89]
Hi 

I've an issue where there is a blue shade appearing on top of my rendered object, I have a feeling it's because jsartoolkit is trying to render the model for every small change and this causes some corruption in the rendering ? 

any help or pointers would be much appreciated .

Thanks 
![screenshot_20180308-113521](https://user-images.githubusercontent.com/751657/37135923-6ef78a56-22c5-11e8-9a34-fd600cbe2d70.jpeg)

**Comment by ThorstenBux** [comment]
That is nothing jsartoolkit related I'm quite sure. I guess your model has a 2nd model underneath it that is blue. Maybe not an entire model but just a blue circle from the look of it and the z-buffer from OpenGL can't decide which model to render on top of the other. inspect your model and see if there is blue geometry in it.

**Comment by siddhartpai** [comment]
Yeah there is a blue geometry on it , But when i load the same model on donmccurdy.gltf-viewer.com , ti loads properly without any artifacts . 
https://i.imgur.com/vgTsX62.png
